GEOSwift/MapKit.swift

```diff
@@ -140,3 +75,29 @@ open class MKShapesCollection: MKShape, MKOverlay {
         }
     }
 }
+
+// MARK: - CLLocationCoordinate2D Conversions
+
+extension CLLocationCoordinate2D {
+    public init(_ coord: Coordinate) {
```

Comment:
These type-conversion initializers seem more idiomatic than the conversion functions we had before. I've moved the old funtions to the bottom of the file and annotated them as unavailable and renamed so that Xcode will provide fixits.